frog/imports/ui/repository/Activity.jsx

```diff
@@ -20,44 +20,46 @@ export default class Activity extends Component {
   }
 
   renderObjectProperties(object, key) {
-        class Property extends Component {
-          render() {
-            return(
-              <div>
-                <br/>
-                <a>{this.props.data.prop + ": " + this.props.data.value}</a>
-              </div>
-            );
-          }
-        }
-
-        return (Object.keys(object).map((prop) =>(
-          <Property key={prop} data = {{prop:prop, value:object[prop]}} />
-        )));
+
+    class Property extends Component {
+      render() {
+        return(
+          <div>
+          <br/>
+          <a>{this.props.data.prop + ": " + this.props.data.value}</a>
+          </div>
+          );
+      }
+    }
+
+    return (Object.keys(object).map((prop) =>(
+      <Property key={prop} data = {{prop:prop, value:object[prop]}} />
+      )));
+
   }
 
 
   render() {
     return (
       <div className="activity-summary">
       <li onClick={this.activityHandler.bind(this)}>
-        {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
+      {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
       </li>
 
       {
         //If the user has clicked on the activity, put prevous properties in addition
         //to hidden properties (returned by the renderObjectProperties)
         this.state.isClicked ?
-          <div className="activity-complete">
-            <br/>
-              {this.props.id}: {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
-              {this.renderObjectProperties(this.props.object, this.props.id)}
-            <br/>
-          </div>
-          : ""
+        <div className="activity-complete">
+        <br/>
+        {this.props.id}: {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
+        {this.renderObjectProperties(this.props.object, this.props.id)}
+        <br/>
+        </div>
```

Comment:
Same here. I disagree with this indentation change. Everything line between `<div>` and `</div>` should have +2 spaces, but it's true that the indents after the `<br/>` was wrong